simulator/src/plotArea.js

```diff
@@ -214,22 +220,18 @@ const plotArea = {
         // Reset canvas
         this.clear();
         var ctx = this.ctx;
-
```

Comment:
Working is fine now 🎉. Just one question, why have you removed this blank line (there are 8-9 more lines you have removed)?"

Reply:
sry for replying so late. Maybe it was some misunderstading from my side. I thought you asked to do so in your last request where you stated to change "unneccesary lines of changes(blanks)"